twoopstracker/twoops/views.py

```diff
@@ -192,6 +192,8 @@ def generate_file(data, filename, fieldnames, fileformat):
 
 class TweetsView(generics.ListAPIView):
     serializer_class = TweetSerializer
+    filter_backends = (filters.OrderingFilter,)
+    ordering_fields = ["owner__screen_name", "deleted_at", "created_at"]
```

Comment:
@esirK how about `number_of_interactions`?

Reply:
@KhadijaMahanga  we decided not to use that because we currently don't have a way to get them.